src/components/user/PeopleTable.tsx

```diff
@@ -445,6 +452,30 @@ const PeopleTable: React.FC<PeopleTableProps> = (props) => {
             />
           </DialogContent>
         </Dialog>
+
+        {!isLargeScreen && (
+          <div className={classes.mobileUpdateButton}>
+            {props.selection && (
+              <ActionButtonContainer>
+                <div className={classes.verticalCentered}>
+                  {selectedParticipants.length} user(s) selected
+                </div>
+                <Button
+                  type="button"
+                  onClick={() => {
+                    if (props.onUpdate) {
```

Comment:
The onClick handler could be moved outside the HTML part into some onClickHandler, and then referenced in both large and small screens.